packages/react-native-worklets/plugin-oxc/src/worklet_string_code.rs

```
@@ -0,0 +1,191 @@
+use oxc_ast::ast::{
+    BindingPattern, Expression, FormalParameters, FunctionBody, FunctionType, PropertyKey,
+    Statement, VariableDeclarationKind,
+};
+use oxc_codegen::{Codegen, CodegenOptions};
+use oxc_span::SPAN;
+
+use crate::transformer::builders::no_rest;
+use crate::utils::rewrite_implicit_return;
+
+
+pub fn build_worklet_body_string<'a>(
```

Clippy (1.97.1)

[warning] this function has too many arguments (11/7)

(warning)